### DISCUSSION TITLE
Expecting easy to understand error messages

### DISCUSSION THREAD

**Discussion by babakfp** [odin-lang/Odin#3336] [General]
Hi 👋

I'm trying out Odin for fun. I got my first error:

![image](https://github.com/odin-lang/Odin/assets/44144724/3830f2d8-4f9e-4fa0-baae-3a3de697c1be)

I don't understand what it says, but I know the problem because I read the docs and single quotes are only for single characters.

I created this to throw it out there so that better error messages may be added in the future.

**Comment by Beefster09** [comment]
Could be cool to add to the error message something like

> Perhaps you meant to make a string literal using double quotes
